rfc6265bis: update HTTP refs to new core specs (#1299)
httpwg/http-extensions: draft-ietf-httpbis-rfc6265bis.md

and WSP (whitespace).

The OWS (optional whitespace) and BWS (bad whitespace) rules are defined in
Section 5.7.3 of {{!HTTPSEM=I-D.ietf-httpbis-semantics}}.

## Terminology

The terms "user agent", "client", "server", "proxy", and "origin server" have
the same meaning as in the HTTP/1.1 specification ({{HTTPSEM}}, Section 3).

The request-host is the name of the host, as known by the user agent, to which
the user agent is sending an HTTP request or from which it is receiving an HTTP
response (i.e., the name of the host to which it sent the corresponding HTTP
request).

> The term request-uri refers to "request target" as defined in Section 6.1.1 of {{HTTPSEM}}.

Reviewer: ```suggestion
The term request-uri refers to "request target" as defined in Section 7.1 of {{HTTPSEM}}.
```

same" matching algorithm for origins are defined in {{RFC6454}}.

"Safe" HTTP methods include `GET`, `HEAD`, `OPTIONS`, and `TRACE`, as defined

> in Section 8.2.1 of {{!HTTPSEM=I-D.ietf-httpbis-semantics}}.

Reviewer: ```suggestion
in Section 9.2.1 of {{HTTPSEM}}.
```

Reviewer: I'm not entirely familiar with the reference syntax here, but this looks like a duplicate inline reference definition?

Origin servers SHOULD NOT fold multiple Set-Cookie header fields into a single
header field. The usual mechanism for folding HTTP headers fields (i.e., as

> defined in Section 5.4.1 of {{HTTPSEM}}) might change the semantics of the Set-Cookie header

Reviewer: ```suggestion
defined in Section 5.3 of {{HTTPSEM}}) might change the semantics of the Set-Cookie header
```

Reviewer: I'm going to merge this one as well since I think it was missed. 5.4.1 doesn't exist in the most recent draft, and the text from the previous draft (https://tools.ietf.org/html/draft-ietf-httpbis-semantics-12#section-5.4.1) is now mostly in 5.3 (https://tools.ietf.org/html/draft-ietf-httpbis-semantics-15#section-5.3).

                    samesite-av / extension-av
expires-av        = "Expires" BWS "=" BWS sane-cookie-date
sane-cookie-date  =

>     <IMF-fixdate, defined in [HTTPSEM], Section 5.7.7>

Reviewer: ```suggestion
    <IMF-fixdate, defined in [HTTPSEM], Section 5.6.7>
```
